### ISSUE TITLE
Options is only binding ID not other attributes

### ISSUE THREAD

**Issue by KumailHussain** [dotansimha/angularjs-dropdown-multiselect#396]
https://jsfiddle.net/michaeldeongreen/22et6sao/9/
you can see this as an example , issue is **option** binds only ID not other attributes like **label**
i want also the label with my model
`<div ng-dropdown-multiselect="" options="example14data" selected-model="example14model" checkboxes="true" extra-settings="example14settings"></div`>
the example14data will show me only the id not the label
why
?????

**Comment by Nithyananadakamath** [comment]
I also have the same issue as @KumailHussain. Adding an option to configure property of selected-model will be helpful.

**Comment by kim0z** [comment]
The same issue, any idea how to save all the selected data and not just the id?

**Comment by kim0z** [comment]
add externalIdProp : '' to extra settings like below:
{enableSearch: true, scrollableHeight: '400px', scrollable: true, externalIdProp : ''}

and it will work

**Comment by Nithyananadakamath** [comment]
Thanks @kim0z, it worked.

**Comment by jeremypeters** [comment]
Yep, same problem here  - setting `displayProp` has no effect. I've tries a few different releases and they all seem to have the same issue.

**Comment by sunkarareddy** [comment]
any idea on how to set selected-model attribute value dynamically during ng-repeat, this really helps. pls suggest.
